### ISSUE TITLE
light_classes.py:450 AttributeError: 'NoneType' object has no attribute 'nodes'

### ISSUE THREAD

**Issue by walles** [jlampel/extra_lights#19]
With Blender 2.82a and Extra Lights 1.0 (according to the Add-ons preferences panel in Blender), if I do...
* Start Blender
* Add a Direct Sunlight
* Check the Create Linked Sky box

... then I get:
```
Traceback (most recent call last):
  File "/Users/johan/Library/Application Support/Blender/2.82/scripts/addons/extra_lights-master/light_classes.py", line 450, in execute
    nodes = world.node_tree.nodes
AttributeError: 'NoneType' object has no attribute 'nodes'

location: <unknown location>:-1

location: <unknown location>:-1
Error: Traceback (most recent call last):
  File "/Users/johan/Library/Application Support/Blender/2.82/scripts/addons/extra_lights-master/light_classes.py", line 450, in execute
    nodes = world.node_tree.nodes
AttributeError: 'NoneType' object has no attribute 'nodes'

location: <unknown location>:-1
```

**Comment by jlampel** [comment]
Hi, sorry I missed this! Thank you for letting me know. It looks like this happens only when no world is active: 
![image](https://user-images.githubusercontent.com/12982300/89925728-a7913c00-dbb8-11ea-9675-d44de824fd9c.png)

This will be fixed in the next update, since I'll be appending the world rather than creating it from scratch.

**Comment by jlampel** [comment]
Fixed in version 1.1
